src/chain/chain.rs

```diff
@@ -481,6 +470,21 @@ impl Chain {
             .should_vote_for_merge(self.elder_size(), self.neighbour_infos())
     }
 
+    /// Increase version by one due to the parsec pruning.
+    /// Should not be called while a pfx change is in progress.
```

Comment:
Probably should not be called anytime churn is in progress since we are already reseting Parsec graph.
Also how is that prevented, do we filter prune votes somewhere?

Reply:
> do we filter prune votes somewhere

when PrefixChange is not None, there will be no vote for ParsecPrune

> Probably should not be called anytime churn is in progress since we are already reseting Parsec graph

yeah, good idea. will think about it.